core/src/com/unciv/ui/options/MultiplayerTab.kt

```diff
@@ -274,3 +283,13 @@ private fun createRefreshOptions(unit: ChronoUnit, vararg options: Long): List<S
         SelectItem(duration.format(), duration)
     }
 }
+
+private fun addSelectAsSeparateTable(tab: Table, settingsSelect: SettingsSelect<*>) {
```

Comment:
Sounds like SettingsSelect should have been conceived as Widget from the start :smiling_imp:  - at least some hint on why wrapping is needed instead of, say, colspan?

Reply:
Nah. "K.I.S.S." - as long as there's thought behind the decisions and not oversight..